AdaptiveCards/templating/language.md

```diff
@@ -14,19 +14,27 @@ Templating enables the separation of **data** from **layout** in your Adaptive C
 
 > [!IMPORTANT] 
 > 
-> These features are **in preview and subject to change**. Your feedback is not only welcome, but  critical to ensure we deliver the features **you** need.
+> **Breaking changes** in the **May 2020 Release Candidate**
+>
+> We've been hard at work getting templating released, and we're finally in the home stretch! We had to make some minor breaking changes as we close on the release.
 
-When authoring a template you can specify the data inline with the `AdaptiveCard` payload, or at runtime using the [Templating SDKs](sdk.md).
+## Breaking changes as of May 2020
```

Comment:
Ideally, these would reference the Release notes which would have the breaking changes listed out to avoid redundancy. Any objection to making that quick change here once we do have the preview github release note published next week?